2022-08-01 15:57:19,724 INFO    Thread-8  :8971 [wandb_setup.py:_flush():76] Configure stats pid to 8971
2022-08-01 15:57:19,724 INFO    Thread-8  :8971 [wandb_setup.py:_flush():76] Loading settings from /home/kana/.config/wandb/settings
2022-08-01 15:57:19,724 INFO    Thread-8  :8971 [wandb_setup.py:_flush():76] Loading settings from wandb/settings
2022-08-01 15:57:19,724 INFO    Thread-8  :8971 [wandb_setup.py:_flush():76] Loading settings from environment variables: {'project': 'noisy_label_cleaning_a', 'entity': 'kka-na', 'root_dir': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script', 'run_id': 'ihmmdsay', 'sweep_param_path': '/home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/sweep-ehlo105g/config-ihmmdsay.yaml', 'sweep_id': 'ehlo105g'}
2022-08-01 15:57:19,724 WARNING Thread-8  :8971 [wandb_setup.py:_flush():76] Could not save program above cwd: ../src/hyp_tuning.py
2022-08-01 15:57:19,724 INFO    Thread-8  :8971 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': None, 'program': '../src/hyp_tuning.py'}
2022-08-01 15:57:19,725 INFO    Thread-8  :8971 [wandb_init.py:_log_setup():461] Logging user logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_155719-ihmmdsay/logs/debug.log
2022-08-01 15:57:19,725 INFO    Thread-8  :8971 [wandb_init.py:_log_setup():462] Logging internal logs to /home/kana/Documents/Training/Reliable_Dataset/cleaning/script/wandb/run-20220801_155719-ihmmdsay/logs/debug-internal.log
2022-08-01 15:57:19,729 INFO    Thread-8  :8971 [wandb_init.py:init():495] calling init triggers
2022-08-01 15:57:19,730 INFO    Thread-8  :8971 [wandb_init.py:init():498] wandb.init called with sweep_config: {'BATCH_SIZE_PER_IMAGE': 64, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0005188156547741864, 'model': None}
config: {}
2022-08-01 15:57:19,732 INFO    Thread-8  :8971 [wandb_init.py:init():548] starting backend
2022-08-01 15:57:19,733 INFO    Thread-8  :8971 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-08-01 15:57:19,742 INFO    Thread-8  :8971 [backend.py:ensure_launched():217] starting backend process...
2022-08-01 15:57:19,749 INFO    Thread-8  :8971 [backend.py:ensure_launched():222] started backend process with pid: 9050
2022-08-01 15:57:19,750 INFO    Thread-8  :8971 [wandb_init.py:init():558] backend started and connected
2022-08-01 15:57:19,766 INFO    Thread-8  :8971 [wandb_run.py:_config_callback():1131] config_cb None None {'BATCH_SIZE_PER_IMAGE': 64, 'IMS_PER_BATCH': 8, 'iteration': 300, 'learning_rate': 0.0005188156547741864, 'model': None}
2022-08-01 15:57:19,769 INFO    Thread-8  :8971 [wandb_init.py:init():634] updated telemetry
2022-08-01 15:57:19,771 INFO    Thread-8  :8971 [wandb_init.py:init():665] communicating run to backend with 30 second timeout
2022-08-01 15:57:20,570 INFO    Thread-8  :8971 [wandb_run.py:_on_init():1919] communicating current version
2022-08-01 15:57:20,608 INFO    Thread-8  :8971 [wandb_run.py:_on_init():1923] got version response 
2022-08-01 15:57:20,608 INFO    Thread-8  :8971 [wandb_init.py:init():700] starting run threads in backend
2022-08-01 15:57:21,708 INFO    Thread-8  :8971 [wandb_run.py:_console_start():1893] atexit reg
2022-08-01 15:57:21,710 INFO    Thread-8  :8971 [wandb_run.py:_redirect():1766] redirect: SettingsConsole.REDIRECT
2022-08-01 15:57:21,710 INFO    Thread-8  :8971 [wandb_run.py:_redirect():1771] Redirecting console.
2022-08-01 15:57:21,710 ERROR   Thread-8  :8971 [wandb_run.py:_redirect():1830] Failed to redirect.
Traceback (most recent call last):
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/wandb_run.py", line 1823, in _redirect
    out_redir.install()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 702, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 652, in add_fd
    self._register()
  File "/home/kana/yes/envs/detectron2/lib/python3.8/site-packages/wandb/sdk/lib/redirect.py", line 637, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/kana/yes/envs/detectron2/lib/python3.8/signal.py", line 47, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread
2022-08-01 15:57:21,711 INFO    Thread-8  :8971 [wandb_init.py:init():732] run started, returning control to user process
2022-08-01 15:57:21,712 INFO    Thread-8  :8971 [wandb_run.py:_finish():1680] finishing run kka-na/noisy_label_cleaning_a/ihmmdsay
2022-08-01 15:57:21,712 INFO    Thread-8  :8971 [wandb_run.py:_atexit_cleanup():1862] got exitcode: 1
2022-08-01 15:57:21,712 INFO    Thread-8  :8971 [wandb_run.py:_restore():1834] restore
2022-08-01 15:57:21,944 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 527
}

2022-08-01 15:57:22,295 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 527
}

2022-08-01 15:57:22,580 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 2820
}

2022-08-01 15:57:22,683 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5909
}

2022-08-01 15:57:22,785 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5909
}

2022-08-01 15:57:22,888 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 527
  total_bytes: 5909
}

2022-08-01 15:57:22,990 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,093 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,195 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,298 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,401 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,503 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,606 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:23,973 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}

2022-08-01 15:57:24,513 INFO    Thread-8  :8971 [wandb_run.py:_on_finish():1991] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5909
  total_bytes: 5909
}
local_info {
}

2022-08-01 15:57:25,657 INFO    Thread-8  :8971 [wandb_run.py:_footer_history_summary_info():3099] rendering history
2022-08-01 15:57:25,657 INFO    Thread-8  :8971 [wandb_run.py:_footer_history_summary_info():3131] rendering summary
2022-08-01 15:57:25,661 INFO    Thread-8  :8971 [wandb_run.py:_footer_sync_info():3055] logging synced files
